src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -155,7 +153,7 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 					self?.showAntigenTestProfile()
 				}
 			},
-			antigenTestProfileStore: antigenTestProfileStore
+			antigenTestProfileStore: store
```

Comment:
We want to reduce the usage of RequiresAppDependencies, because its tight coupling.
The injection of antigenTestProfileStore allows us to provide different implementations at runtime.

Reply:
This was not used in any tests. Should we still leave it?

Reply:
Yes, we want to get rid of RequiresAppDependencies, so we should not use it.

Reply:
Done